### DISCUSSION TITLE
Importar Dados de Sistemas

### DISCUSSION THREAD

**Discussion by RFS-Solutions** [RamonSilva20/mapos#2577] [General]
É possível importar dados de Clientes, OS, etc... de outro sistema?

**Comment by lukasabino** [comment]
Bom dia! 

Sim, é possível, porém tem que fazer na mão, se o sistema doador das informações exportar em XLSX, dá uma boa ajuda, depois, só adequar com as colunas do banco de dados do Map-Os e importar direto no banco do sistema.
